src/site/assets/js/static-page-widgets.js

```diff
@@ -1,49 +1,48 @@
-function mountWidgets(widgets, isProduction) {
-  widgets
-    .filter(function (widget) {
-      if (widget.production === false && isProduction) {
-        return false;
-      }
-
-      return true;
-    })
-    .forEach(function (widget) {
-      var root = document.getElementById(widget.root);
-      var timeout = (widget.timeout || 0) * 1000;
-      var slowLoadingThreshold = (widget.slowLoadingThreshold || 6) * 1000;
-      var slowMessage = widget.slowMessage || 'Sorry, this is taking longer than expected.';
-
-      if (!widget.showSpinnerUnauthed) {
-        root.innerHTML = '<div class="loading-indicator-container">' +
-          '<div class="loading-indicator" role="progressbar" aria-valuetext="' + widget.loadingMessage + '"></div>' +
-          '<span class="loading-indicator-message">' + widget.loadingMessage + '</span>' +
-        '</div><span class="static-content"/>';
+function mountWidgets(widgets, slowLoadingThreshold) {
+  [].slice
+    .call(widgets)
```

Comment:
this is interesting- is this a typecast from something like a Nodelist?

Reply:
Yeah, this is the pre-`Array.from` method of converting a NodeList to a regular array. This file isn't run through Babel, so it needs to work in IE 11 as is.